CommandHelper/LocalPackages/command vegetasupersaiyanbeyondblue.yml

```diff
@@ -0,0 +1,40 @@
+alts:
```

Comment:
His file name is way wrong.

What it should be:
CommandHelper/LocalPackages/commands/chars/vegetasupersaiyanbeyondblue.yml

What you have:
CommandHelper/LocalPackages/command vegetasupersaiyanbeyondblue.yml